src/WebJobs.Script/Binding/FunctionBinding.cs

```diff
@@ -48,6 +48,7 @@ internal static Collection<FunctionBinding> GetBindings(ScriptHostConfiguration
                     string type = function.Type.ToLowerInvariant();
                     switch (type)
                     {
+                        // TODO: remove this
```

Comment:
I'll clean this up in phase 2. I can remove this comment from this PR if needed.